java/arcs/core/host/AbstractArcHost.kt

```diff
@@ -356,7 +357,7 @@ abstract class AbstractArcHost(vararg initialParticles: ParticleRegistration) :
         handleSpec: Plan.HandleConnection,
         holder: HandleHolder
     ) = when (handleSpec.type) {
-        is SingletonType<*> ->
+        is SingletonType<*>, is EntityType ->
```

Comment:
I just updated recipe2plan; there should no longer be raw `EntityType`s. They will wrapped with either `SingletonType` or `CollectionType`.